pom.xml

```diff
@@ -1966,8 +1965,6 @@ flexible messaging model and an intuitive client API.</description>
         <test.additional.args/>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
-        <pulsar.broker.compiler.release></pulsar.broker.compiler.release>
```

Comment:
This should be done intentionally.

Reply:
@abhilashmandaliya to fix IDE warning, you can collapse the tag instead of removing it.

Anyway, keep one PR for one purpose (upgrade kotlin lib) can help you get a smooth review experience - we don't distract attention.

Reply:
Thanks, @nodece and @tisonkun.  Have reverted that change.